upload/common/ingest_notifier.py

```diff
@@ -1,74 +1,107 @@
 import json
 import os
 import uuid
-import pika
 import requests
+import jwt
+import time
+import base64
+
+from tenacity import retry, stop_after_attempt, wait_fixed
 
 from .exceptions import UploadException
 from .logging import get_logger
 from .database import UploadDB
+from .upload_config import UploadConfig, UploadAuthConfig
 
 logger = get_logger(__name__)
 
 
 class IngestNotifier:
 
-    FILE_UPLOAD_EXCHANGE = 'ingest.file.staged.exchange'
-    FILE_UPLOADED_QUEUE = 'ingest.file.create.staged'
-
-    FILE_VALIDATED_EXCHANGE = "ingest.validation.exchange"
-    FILE_VALIDATED_QUEUE = "ingest.file.validation.queue"
+    FILE_UPLOADED_ENDPOINT = "messaging/fileUploadInfo"
+    FILE_VALIDATED_ENDPOINT = "messaging/fileValidationResult"
```

Comment:
could be a dict:
```
    {
        "file_uploaded": "messaging/fileUploadInfo",
        "file_validated": "messaging/fileValidationResult"
    }
```

Then index into the dict in __init__

Reply:
Allows for deletion of a few lines of code. Will incorporate